xsl/data-to-json.xsl

```diff
@@ -38,7 +38,7 @@
     <xsl:text>",</xsl:text>
 
     <xsl:text>"t": "</xsl:text>
-    <xsl:value-of select="dita-ot:escapeQuote(@title)"/>
+    <xsl:value-of select="replace(dita-ot:escapeQuote(@title), '\s+', ' ')"/>
```

Comment:
For consistency this should be `normalize-space(...)`

Reply:
Agreed and updated with latest commit.